src/ralph/ssl_certificates/management/commands/update_service_env.py

```diff
@@ -0,0 +1,58 @@
+import sys
+
+from django.core.management.base import BaseCommand
+
+from ralph.assets.models.assets import ServiceEnvironment
+from ralph.dns.dnsaas import dnsaas_client
+from ralph.ssl_certificates.models import SSLCertificate
+
+
+class Command(BaseCommand):
+    help = 'Checks the compliance of services in SSL Certificates'
+
+    def update_from_record(self, results):
+        for value in results:
+            domain = value['content']
+            service_dns = value['service_name']
+            name = None
+            try:
+                name = ServiceEnvironment.objects.get(
+                    service__name=service_dns, environment__name='prod'
+                )
+            except ServiceEnvironment.DoesNotExist:
+                self.stderr.write(
+                    'Service with name {} does not exist'.format(service_dns)
+                )
+            SSLCertificate.objects.filter(
+                domain_ssl=domain).update(service_environment=name)
+
+    def update_from_domains(self, results):
+        for value in results:
+            domain = value['name']
+            service_dns = value['service_name']
+            name = None
+            try:
+                name = ServiceEnvironment.objects.get(
+                    service__name=service_dns, environment__name='prod'
+                )
+            except ServiceEnvironment.DoesNotExist:
+                self.stderr.write(
+                    'Service with name {} does not exist'.format(service_dns)
+                )
+            SSLCertificate.objects.filter(
+                domain_ssl=domain).update(service_environment=name)
+
+    def get_records(self):
+        url = dnsaas_client.build_url('records')
```

Comment:
are you interested here in all types of records (A, TXT, CNAME, PTR) or some particular?

Reply:
In all types of records